arroyo/processing/strategies/run_task.py

```diff
@@ -13,6 +13,37 @@
 TResult = TypeVar("TResult")
 
 
+class RunTask(ProcessingStrategy[TPayload]):
+    """
+    Basic strategy to run a custom processing function on a message.
+    """
+
+    def __init__(
+        self,
+        function: Callable[[Message[TPayload]], TResult],
```

Comment:
Food for thoughts, now that we have a class that represents the payload of a message, should we change the interface of the transform function to take a value in and spit a value out  for consistency ?

Reply:
What's the rationale for returning a value instead? This way the transform function is the same as it was before... and it's probably easier for the `value.replace(x)` call to be done here by the transform step instead of every user needing to call it by themselves.

Reply:
I was more interested in passing the value instead of the message. 
It is just about a more intuitive api. A transformation function takes a an object and returns an objects that has the same role (thus Value -> Value)

Reply:
I see. I guess that's possible. I don't think the decision here is related to the changes to Value/Message that we introduced that much though, or that the calculus really changes now. Like, it could just as easily have been Message -> Message before but we decided not to?

Reply:
>  it could just as easily have been Message -> Message before but we decided not to?

I cannot remember what the reason was.